src/mds/Server.cc

```diff
@@ -463,8 +463,13 @@ void Server::reclaim_session(Session *session, const cref_t<MClientReclaim> &m)
     finish_reclaim_session(session, reply);
     return;
   }
-
-  ceph_abort();
+  if(m->get_connection()->get_peer_type() == CEPH_ENTITY_TYPE_CLIENT) {
+    derr << "Server received unknown message " << m->get_type() << ", closing session and blocklisting the client " << session->get_client() << dendl;
```

Comment:
Here IMO you should print the unknown reclaim flag bit instead, it's not a unknow message type, just an unknown flag bit.

Reply:
Right, will make the change.